cif_pow.dic

```diff
@@ -11513,6 +11513,55 @@ save_PD_QPA_INTENSITY_FACTOR
          '_pd_qpa_intensity_factor.diffractogram_id'
          '_pd_qpa_intensity_factor.phase_id'
 
+    _description_example.case
+;
+         data_phase_1
+             _pd_phase.id   PHASE_A
+             _pd_qpa_calib_factor.I_over_Ic   3.26
+
+         data_phase_2
+             _pd_phase.id   PHASE_B
+             _pd_qpa_calib_factor.I_over_Ic   4.79
+
+         data_phase_3
+             _pd_phase.id   PHASE_C
+             _pd_qpa_calib_factor.I_over_Ic   1.00
+
+         data_diffractogram_block
+             _pd_diffractogram.id   UNKNOWN_DIFFRACTOGRAM
+
+             loop_
+             _pd_phase_mass.phase_id
+             _pd_phase_mass.percent
+             PHASE_A   52.02(15)
+             PHASE_B   17.90(15)
+             PHASE_C   30.08(14)
+
+             _pd_qpa_overall.method   I/Ic
+
+             loop_
+             _pd_qpa_intensity_factor.phase_id
+             _pd_qpa_intensity_factor.value
+             PHASE_A   242.54(81)
+             PHASE_B   122.6(12)
+             PHASE_C    43.02(25)
+;
+    _description_example.detail
+;
+         A diffraction pattern containing three phases (PHASE_1, PHASE_2, and
```

Comment:
I brought up a similar point in another review, but the listed phases (PHASE_1, PHASE_2, NIST SRM646a) do not directly map neither to the data names (phase_1, phase_2, phase_3) nor to the _pd_phase.id (PHASE_A, PHASE_B, PHASE_C). It might be nicer if it would.

Reply:
Yeah. I don't know how that wasn't already there. Must have slipped during the various iterations. Will change